roles/rsyslog/vars/outputs/files/main.yml

```diff
@@ -19,11 +19,15 @@ __rsyslog_conf_files_output_modules:
     type: 'modules'
     sections:
 
+      # yamllint disable rule:line-length
       - comment: 'Log messages into files with timestamp format either
```

Comment:
does the choice of `logging_files_template_format` affect only the timestamp format, or does it affect other fields as well?

Reply:
Based on the doc https://www.rsyslog.com/doc/v8-stable/configuration/templates.html, it looks `RSYSLOG_SyslogProtocol23Format` affects other fields.
```
template(name="RSYSLOG_SyslogProtocol23Format" type="string"
     string="<%PRI%>1 %TIMESTAMP:::date-rfc3339% %HOSTNAME% %APP-NAME% %PROCID% %MSGID% %STRUCTURED-DATA% %msg%\n")
```

Reply:
ok - then I guess change the comment to something like "Log messages into files with traditional, syslog, or default format" - or something like that

Reply:
Thank you, @richm. Fixed.